dcp/DcpDriver.go

```diff
@@ -285,9 +286,19 @@ func (d *DcpDriver) reportError(err error) {
 	utils.AddToErrorChan(d.errChan, err)
 }
 
+func allowedCompletionError(err error) bool {
+	switch err {
+	case gocbcore.ErrStreamClosed:
+		return true
+	default:
+		return false
+	}
+}
+
 func (d *DcpDriver) handleVbucketCompletion(vbno uint16, err error, reason string) {
-	if err != nil {
-		d.reportError(err)
+	if err != nil && !allowedCompletionError(err) {
```

Comment:
is it either err OR allowedCompletionError or err AND allowedCompletionError?

Reply:
This means if the error exists and it's considered a fatal error